utils/_context/_scenarios.py

```diff
@@ -519,7 +521,19 @@ def post_setup(self):
 
         elif self.use_proxy:
             self._wait_interface(interfaces.library, self.library_interface_timeout)
-            self.weblog_container.stop()
+
+            if self.library in ("nodejs",):
```

Comment:
@cbeauchesne why not do if else here with _wait_interface ?

Reply:
answer: because some tests might want to wait before flushing (time based tests)